README.md

```diff
@@ -346,6 +346,13 @@ Logical volume size can be extended, but not reduced -- this is for
 safety, as manual intervention is probably required for data
 migration, etc.
 
+## Deprecation Notice
+
+Some facts reported by this module are being deprecated in favor of upcomming structured facts.  The following facts are being deprecated:
```

Comment:
The below facts are the custom flat facts, so not sure if these are gonna deprecate as well?
AFAIK the custom facts (irrespective of flat or structured facts) these will not be deprecated. Only the [legacy facts](https://www.puppet.com/docs/puppet/7/core_facts.html#legacy-facts) which will get deprecated.

Reply:
Facter code generating fact names dynamically on the fly is problematic: violates coding rules, hard to maintain, even harder to test. That's why we should get rid of them.